redash/utils/__init__.py

```diff
@@ -186,3 +186,12 @@ def base_url(org):
 
 def filter_none(d):
     return select_values(lambda v: v is not None, d)
+
+
+def to_filename(s):
```

Comment:
I'd get rid of all comments here. Code is self explanatory and comments will get stale. They are already lying 😝